### ISSUE TITLE
Choose a Non-Sql database for this project

### ISSUE THREAD

**Issue by h3ct0rjs** [librebeer/FlaskMapsPereira#1]
It will include all the operations with the database.

**Comment by alsuga** [comment]
what about Mongo? or try another database?

**Comment by h3ct0rjs** [comment]
Yeah I think that we can use pymongo  for all the connections , Aproved, I  need to understand how generate API's rest using flask.

**Comment by ivanchoff** [comment]
is this project only the API REST? it would be better if you add a readme file with the main idea of project
